### DISCUSSION TITLE
Use only readme on front page

### DISCUSSION THREAD

**Discussion by mister-ben** [ankitskvmdam/clean-jsdoc-theme#174] [Q&A] [answered]
Currently only the home page the readme specified in .jsdoc/config.json is displayed followed by a listing of every js file, each in a h1. It's too much for the start page, and redundant with the sidebar nav. Can we disable this list and show only the readme?

**Comment by mister-ben** [comment]
I see this behaviour is not configurable, at https://github.com/ankitskvmdam/clean-jsdoc-theme/blob/d89b68d9b69cf12ddf96da55a80fe246a6568f97/publish.js#L908

**Comment by ankitskvmdam** [comment]
I am not able to understand tha question. Could you please rephrase it?

In homepage we are only showing the content of the Readme file. And the list of classes, modules, etc are shown on the sidebar.

**Reply by mister-ben** [reply]
I'm referring to the list of files on the front page visible [here](https://deploy-preview-8048--videojs-docs.netlify.app/). I want only the readme content on that page.

**Reply by ankitskvmdam** [reply]
Hmm... this shouldn't be the case. It supposes to use the Readme file only. Could you please share the jsdoc config file? If possible please share a repo to reproduce the same you shared here.

**Reply by mister-ben** [reply]
Thanks, it's [here](https://github.com/videojs/video.js/blob/d9bf1eab903dfe2da4e7771a66336a8b9ee15344/.jsdoc.json). That build is generated from [this PR](https://github.com/videojs/video.js/pull/8048). And thanks for the theme, it looks great.

**Reply by ankitskvmdam** [reply]
Thanks for reporting this. I will look into it. It definitely does not look good to have file names listed on the main page.

I will keep you updated.

**Reply by ankitskvmdam** [reply, marked as answer]
@mister-ben 
It is happening because of the `@file` tag (https://jsdoc.app/tags-file.html) at the beginning of all the js files in your repo.
To solve:
 1. You can remove all the `@file` tags. Example: [big-play-button.js](https://github.com/videojs/video.js/blob/main/src/js/big-play-button.js#L2)
 2. Upgrade to v4.2.0. I have released `clean-jsdoc-theme: 4,2,0`.  Now by default file list is not included on the main page.

**Reply by mister-ben** [reply]
Thanks very much for the quick response. That's working as I want in 4.2.0. I've run into a new issue with `include_css` in this version however; I'll open a new issue/PR for that.

**Comment by ankitskvmdam** [comment]
In v5 the README is the home page by default. Closing as resolved. Docs: https://ankdev.me/clean-jsdoc-theme/
